packages/plugin-react/src/index.ts

```diff
@@ -131,6 +131,12 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
     name: 'vite:react-babel',
     enforce: 'pre',
     config() {
+      if ('rolldownVersion' in vite) {
```

Comment:
We should check for babel plugins, some people may depend on some weird legacy things that will not be supported by oxc

Reply:
I've checked for the whole `babel` option. Do you think that is good enough or should I check for `function` _or_ `babel.plugins` being populated?

Reply:
I think we also need to skip if there's any plugin that has `api.reactBabel` as that may update the babel option.
https://github.com/vitejs/vite-plugin-react/blob/7679e49bf4433afe13e7df91a0f2e498720d29db/packages/plugin-react/src/index.ts#L189-L191

Reply:
Checked for hooks.length 👍